src/app/public/modules/radio/radio.component.ts

```diff
@@ -110,6 +147,15 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     return this._name;
   }
 
+ // tslint:disable: max-line-length
+/**
+ * This property is deprecated in favor of
```

Comment:
This link doesn't work for me in the preview. We can probably remove the link for now.